app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -50,7 +54,11 @@ ns.SearchPage = do (ko
     $('.launch-customize-modal').click ->
       $('#customizeDataModal').modal('show')
 
-    preferences.ready.done(-> initModal()) if !window.edscportal
+    reverb = ["http://echo-reverb-rails.dev", "https://testbed.echo.nasa.gov", "https://api-test.echo.nasa.gov", "https://testbed.echo.nasa.gov", "https://reverb.echo.nasa.gov"]
```

Comment:
Move everything related to showing modal, setting the cookie and writing back the metrics here inside ```$(document).ready``` so that they are performed only after user sees the page.